spark/src/main/scala/org/apache/spark/sql/delta/skipping/MultiDimClusteringFunctions.scala

```diff
@@ -54,4 +56,24 @@ object MultiDimClusteringFunctions {
   def interleave_bits(cols: Column*): Column = withExpr {
     InterleaveBits(cols.map(_.expr))
   }
+
+  /**
+   * Transforms the provided integer columns into their corresponding position in the hilbert
+   * curve for the given dimension.
+   * @see http://www.dcs.bbk.ac.uk/~jkl/thesis.pdf
```

Comment:
link has rotted; https://citeseerx.ist.psu.edu/document?repid=rep1&type=pdf&doi=bfd6d94c98627756989b0147a68b7ab1f881a0d6 seems to be equivalent

Reply:
Thanks! Fixed.